### PR TITLE
Add additional condition to power on comm radios.

### PR DESCRIPTION
**Pull request by Simworld2020**
### Description of Changes
Apologies for git errors, new to git, I modified the PULL_REQUEST_TEMPLATE.md file which contains the text of my pull request.
I know better now, thank you.

### Checklist:

* [x] My changes follow the Contributing Guidelines. 
* [x] My changes implement realistic features. 
* [x] Please have a main Developer test my changes before merging.

### CODE REVIEW

**Comment by merspieler** [comment]
> Apologies for git errors, new to git, I modified the PULL_REQUEST_TEMPLATE.md file which contains the text of my pull request.
I know better now, thank you.

You don't need to modify that file... you will get its content, where you've added this comment... there you can modify it, without changing that template... so please revert that commit.... else the next person who wants to open a PR will get your text ;-)

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
-<!-- Give a brief summary of your changes in the title. -->
+To add an additional condition to bring voltage to comm radios.
 
 ### Description of Changes
-<!-- Describe your changes in detail. -->
-
+*I do not have airbus specs for the navcom radio control panels, but I believe the following:*
```
Comment (merspieler) [inline PULL_REQUEST_TEMPLATE.md]: We DO NOT go by believe... we always go by the manual or if that isn't helping us, by reports from pilots and videos of the real aircraft.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
+There are three navcom radio control panels and each has an on/off switch which stands between the voltage source and the navcom radio stack.
+The three control panels are wired in parallel, at least one of the three switches must be 'on' for the dc-ess voltage to reach
+the navcom radio stack.
+If anyone has an authoritative, study level resource on how the radio control panels and audio panels work I would very much appreciate receiving
```
Comment (merspieler) [inline PULL_REQUEST_TEMPLATE.md]: Ping me on the forum, and I can share the manuals with you.

**Review by merspieler — CHANGES_REQUESTED** [review]
Please:
* Revert the pull request template change and write your description in the pull request on github
* According to FCOM DSC-23-10-50 P 1/2, VHF(1,2,3) are listed independently from RMP(1,2,3) which means they are independently powered.

**Comment by legoboyvdlp** [comment]
Hi, sorry, I disagree with this pull request.

The manuals are clear that the RMP switch is merely for RMP power.

They state that: "ON/OFF sw
**This switch controls the power supply to the RMP.**
Note: RMP 3 is able to control VHF and HF transceivers through RMP 1 and RMP 2 even
when they are OFF."

I therefore conclude that the fact you can control VHF1 or VHF3 through RMP3 even if the other RMPs are off, means that the power supply to the radios is independent. If RMP1 is switched off, if you have DC ESS power, VHF-1 will still work -- and you can operate VHF1 through any other RMP. If all three RMPs are switched off, the radio will still work - but you can't control it anymore.

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Simworld2020) [reply PULL_REQUEST_TEMPLATE.md]: Okay, I've reverted the commits, will request documents from you in the forum and thank you.

**Comment by Simworld2020** [comment]
Okay I have reverted the commits and will reach out to merspieler in the forum to get the reference documents, thank you both.

**Pull request closed without merging**